src/integrator.jl

```diff
@@ -9,49 +14,52 @@ function (::Leapfrog)(ϵ::AbstractFloat)
     return Leapfrog(ϵ)
 end
 
-function is_valid(v::AbstractVector{<:Real})
-    if any(isnan, v) || any(isinf, v)
-        return false
-    end
-    return true
-end
-
-function lf_momentum(ϵ::T, h::Hamiltonian, θ::AbstractVector{T}, r::AbstractVector{T}) where {T<:Real}
+function lf_momentum(ϵ::T,
```

Comment:
see, http://turing.ml/docs/style-guide/#method-definitions

Reply:
OK then we shouldn't reformat like this right?

Reply:
yes; because if we break out of the loop when `i = 1`, variables `θ` and  `r` are not updated at all, see

https://github.com/TuringLang/AdvancedHMC.jl/blob/10659b225f5ddc8c4614cdd62cc4bc27799c10a8/src/integrator.jl#L45-L47